docs-source/usersguide/application/02_running_sims.rst

```diff
@@ -593,6 +593,40 @@ such as :file:`charmm36/water.xml`, which specifies the default CHARMM water mod
 The converted parameter sets come from the `CHARMM36 July 2017 update <http://mackerell.umaryland.edu/charmm_ff.shtml>`_
 and were converted using the `openmm-forcefields <https://github.com/choderalab/openmm-forcefields>`_ package and `parmed <https://github.com/parmed/parmed>`_.
 
+Implicit Solvent
+----------------
+
+The Amber and CHARMM force fields described above can be used with any of the Generalized
+Born implicit solvent models from AMBER.  To use them, include an extra file when
+creating the ForceField.
+
+.. tabularcolumns:: |l|L|
+
+======================  ==================================================================================================================================
+File                    Implicit Solvent Model
+======================  ==================================================================================================================================
+:file:`implicit/hct`    Hawkins-Cramer-Truhlar GBSA model\ :cite:`Hawkins1995` (corresponds to igb=1 in AMBER)
```

Comment:
The ffxml files we list in tables above end in `.xml`---e.g. `charmm36.xml` on line 539. Should we do the same here for consistency?

Reply:
Oops, thanks for catching that.